### ISSUE TITLE
DDC commands not being issued reliably when building with Xcode > 11.3.1

### ISSUE THREAD

**Issue by JoniVR** [MonitorControl/MonitorControl#238]
On recent versions of Xcode, I get some reliability issues with DDC in MonitorControl. I'm not sure why this is happing but I'm wondering if anyone else is running into this too.

Note: I haven't tested all the versions after 11.3.1 but current last version (11.5 and the one version below that don't work for me).

**Comment by legendaryaman** [comment]
I can confirm that keys are not working on **Xcode 11.6**, although works fine on Xcode 11.3.1.

**Comment by JoniVR** [comment]
Glad to know someone else noticed this too. Commands are working intermittently but not consistently. Should really test if it's a DDC.swift issue.

**Comment by JoniVR** [comment]
I just went back to test some more, DDC commands are not being issues consistently when using keys on version 11.4 and up. I'm pretty sure commands are being sent to DDC but the display values are not always being updated. 

@reitermarkus, could you please test if this also happens for you if you have time? I ran the tests on your DDC.swift repo and everything seems fine, commands mostly work when dragging using the slider. 
Using shortcuts or clicking in the slider to instantly change the value doesn't work.

**Comment by JoniVR** [comment]
DDC.swift issue can be resolved by including the DDC.swift framework manually and then removing the `serialNumber` function from `EDID+Extension.swift` as it seems to conflict somewhere. Will dig into why this is happening soon but at least we have a general idea of what it could be.

**Comment by Takuro-Ito** [comment]
> commands mostly work when dragging using the slider.
> Using shortcuts or clicking in the slider to instantly change the value doesn't work.

I just found out that `func valueChanged(slider: NSSlider)` in `SliderHandler.swift` is called multiple times (mostly 3 to 5 times for dragging).
So I modified code in `override func stepBrightness(isUp: Bool, isSmallIncrement: Bool)`  in `ExternalDisplay.swift` like this then it works fine.
```swift
// Send DDC command at least 3 times to get consistent result with my environment.
for _ in 0 ..< 3 {
  _ = self.ddc?.write(command: .brightness, value: ddcValue)
}
```
But I don't think this is a legitimate solution and don't know why Xcode built `DDC.swift` works fine without sending commands multiple times on a key tap.
Any ideas? @JoniVR @reitermarkus
